CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 4.8.0
+* Plumb platformInfo in Configuration for PHC use (#1757) via Joshua Liebowitz (@taquitos)
+* `EntitlementInfo`: added `isActiveInCurrentEnvironment` and `isActiveInAnyEnvironment` (#1755) via NachoSoto (@NachoSoto)
+* `logErrorIfNeeded`: also log message if present (#1754) via NachoSoto (@NachoSoto)
+* `bug_report.md`: clarify SK2 support (#1752) via NachoSoto (@NachoSoto)
+* Re-fetch cached offerings and products after Storefront changes (3/4)  (#1743) via Juanpe Catalán (@Juanpe)
+* Prepare next version: 4.8.0-SNAPSHOT (#1750) via RevenueCat Releases (@revenuecat-ops)
+* Fix replace version without prerelease modifiers (#1751) via Toni Rico (@tonidero)
+* added a log when `autoSyncPurchases` is disabled (#1749) via aboedo (@aboedo)
```

Comment:
Can we split this into "new APIs", "bug fixes" and "other"? 
We need to automate the process, but for now, it'd be nice to keep the changelogs nice

Reply:
<YouGotItDude.gif>

Reply:
✅